### PR TITLE
[OYPD-233] Can't edit user menu

### PR DESCRIPTION
**Pull request by hamrant**
Jira - https://propeople-us.atlassian.net/browse/OYPD-233
d.org issue - https://www.drupal.org/node/2859195

Steps for review:
- [x] login as admin
- [x] Click on Pencil icon on user menu
- [x] Check that you can see contextual links
- [x] profit

### CODE REVIEW

**Comment by gundevel** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build711 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build711 |
| Installation process | http://install.openy-dev.ffwua.com/build711/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/711/

**Comment by gundevel** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build711/profiles/openy/build/reports/behat

File: themes/openy_themes/openy_rose/openy_rose.info.yml [hunk 1]
```diff
@@ -11,6 +11,7 @@ regions:
   logo: Logo
   primary_menu: Primary Menu
   secondary_menu: Secondary Menu
+  sidebar: Sidebar
```
Comment (alexschedrov) [inline themes/openy_themes/openy_rose/openy_rose.info.yml]: Let's call it mobile sidebar.

**Review by alexschedrov — CHANGES_REQUESTED** [review]

**Comment by bjwilkins** [comment]
Build comment file: 
--------------
===============
 
| Environment | Link |
| ------------- | ------------- |
| Fresh OpenY installation | http://openy-dev.ffwua.com/build713 |
| Upgraded(upgrade path) installation | http://upgrade.openy-dev.ffwua.com/build713 |
| Installation process | http://install.openy-dev.ffwua.com/build713/install.php |

--------------
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://openy-dev.ffwua.com:8080/job/PR_BUILDER/713/

**Comment by bjwilkins** [comment]
Behat test results file URL: http://openy-dev.ffwua.com/build713/profiles/openy/build/reports/behat

**Review by alexschedrov — APPROVED** [review]
Approved.

**Pull request merged**